content/reduced-form-specification.md

```diff
@@ -29,144 +68,283 @@ Some datasets (such as [HadEX2](https://climatedataguide.ucar.edu/climate-data/h
+is being consideration. For example, doing
+aggregation-before-transformation for temperature will distort the
+signal less that doing it for precipitation. This is because
+precipitation is highly local both temporally and spatially; it could
+rain for <1 min in <1 km radius area.
 
 ### Transformation-before-aggregation
 
-When an economic process is occurring at the grid level (for example, for individuals or households), we need to first do our estimation at the grid level. For example, to estimate the effect of temperature on human mortality at the county level, we should reckon that the effect of temperature on mortality is a local phenomenon, so the estimation should happen at the lowest possible level. Here, we need to do the required transformation of our weather variables at the grid level, then aggregate these values using a weighted averaging method, and feed these into our estimation procedure.
+When an economic process is occurring at the local level (for example,
+for individuals or households), we need to first do our estimation at
+the grid level. For example, to estimate the effect of temperature on
+human mortality at the county level, we should reckon that the effect
+of temperature on mortality is a local phenomenon, so the estimation
+should happen at the lowest possible level.  Since the dependent
+variable is a sum of individual-level outcomes, we should write down
+the reduced-form specification for an individual experiencing
+high-resolution weather, and then sum across all of those reduced
+forms. The result is that we need to do any non-linear transformation of our weather variables at the grid level, then aggregate these values using a weighted averaging method, and feed these into our estimation procedure.
 
 **Mathematical formulation for transformation-before-aggregation method**
 
 We want to understand how local agents respond to weather shocks. Suppose that there exists an agent-level dose-response curve, $y_{js} = f(T_{ps})$, for a socioeconomic outcome for agent $j$, where the temperature affecting agents is in grid cell $p$ and occurs in timestep $s$ (e.g., if the agents respond on a day-by-day basis, $T_{ps}$ is the local weather for a single day).  
 
 However, we do not observe agent-level responses. Instead, we have region-wide sums, $y_{it}$ for region $i$ and reporting period $t$. For example, if $y_{js}$ is death risk for agent $j$ for a given day, we may only observe total deaths across a region in each year, $$y_{it} = \sum_{s \in t} \sum_{j \in i} y_{js}.$$  
 
-We can determine the agent-level response $f(T_{ps})$ if we assume linearity. First, let us represent this the way we would if we could run a regression with agent-level data, breaking up the dose-response
+We can determine the agent-level response $f(T_{ps})$ if we assume
+linearity. First, let us represent this the way we would if we could run a regression with agent-level data, breaking up the dose-response
```

Comment:
replace "let us represent this the way we would if we" with "let us represent this as if we"